etcdctlv3/command/watch_command.go

```diff
@@ -110,10 +110,13 @@ func recvLoop(wStream pb.Watch_WatchClient) {
 		switch {
 		// TODO: handle canceled/compacted and other control response types
 		case resp.Created:
+			fmt.Printf("[revision: %d]\n", resp.Header.Revision)
```

Comment:
I do not think we should print out header by default. Probably should be in extensive mode or something.